test/Garnet.test/RespVectorSetTests.cs

```diff
@@ -1481,6 +1481,64 @@ public void RecreateIndexesOnRestore()
 
         // TODO: FLUSHDB needs to cleanup too...
 
+        [Test]
+        public void VINFO()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase();
+
+            // Create a vector set with known parameters
+            var res1 = db.Execute("VADD", ["foo", "REDUCE", "5", "VALUES", "10", "1.0", "2.0", "3.0", "4.0", "5.0", "6.0", "7.0", "8.0", "9.0", "10.0", new byte[] { 0, 0, 0, 0 }, "Q8", "EF", "16", "M", "32"]);
+            ClassicAssert.AreEqual(1, (int)res1);
+
+            // Get VINFO - should return an array of 12 elements (6 key-value pairs)
+            var res2 = (RedisValue[])db.Execute("VINFO", ["foo"]);
+            ClassicAssert.AreEqual(12, res2.Length);
```

Comment:
This makes an assumption about ordering, probably map these into a dictionary and then validate so ordering isn't a hard assumption.